### ISSUE TITLE
validateAccessWidener should log all errors

### ISSUE THREAD

**Issue by Technici4n** [FabricMC/fabric-loom#895]
When updating to a new MC version for example, it would be helpful to show all of the AW errors, so they could be fixed in a single pass.

**Comment by modmuss50** [comment]
Yeah, this would be nice, its somewhat hard to do due to AWs design while being able to report the line number. Will likely be something that needs fixing in CT.
